File: Bimestre 2/Java/Arreglos/src/arreglos/VectorParImp.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package arreglos;

import java.util.Scanner;

/**
 *
 * @author diegojp
 */
public class VectorParImp {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        int n,i,aux;
        n=0;
        
        System.out.println("Ingrese el tamaño del vector");
        n=leer.nextInt();
        System.out.println("----------------------------");
        int [] A = new int [n];
        int [] par = new int [n];
        int [] impar = new int [n];
        for(i=0; i<n; i++){
            System.out.println("Ingrese los terminos del vector");
            A[i]=leer.nextInt();
        
        }
        //Presentar vector original
        System.out.println("Vector original");
        for(i=0; i<n; i++){
            System.out.println("A["+i+"]="+A[i]);
        }
        for(i=0; i<n; i++){
                if(A[i]%2==0){
                aux=par[i];
                par[i]=A[i];
                System.out.println("Pares "+i+" : "+par[i]);
                
                
                
                }
                
        }
        System.out.println("------------------");
        for(i=1; i<n; i++){
            if(A[i]%2!=0){
                impar[i]=A[i];
                aux=impar[i];
                System.out.println("Impares "+i+" : "+impar[i]);
                
                }
        }

        
        
    }
}

File: Bimestre 2/Python/Arreglos/VectorParImp.py
n=0;

print("Ingrese el tamaño del vector")
n=int(input())
print("----------------------------")
A = []
par = []
impar = []
for i in range(n):
    print("Ingrese los terminos del vector")
    num=int(input())
    A.append(num)
#Presentar vector original
print("Vector original");
for i in range(n):
    print("A[",i,"]=",A[i]);

for i in range(n):
    if(A[i]%2==0):
        aux=par[i]
        par[i]=A[i]
        print("Pares ",i," : ",par[i])

print("------------------")
for i in range(n):
    if(A[i]%2!=0):
        impar[i]=A[i]
        aux=impar[i]
        print("Impares ",i," : ",impar[i])
